.github/workflows/check-pull-request.yml

```diff
@@ -84,3 +103,40 @@ jobs:
           key: ${{ runner.os }}-cargo-test-${{ matrix.runtime }}-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: ${{ runner.os }}-cargo-test-${{ matrix.runtime }}-
       - run: cargo test --all-features --all-targets ${{ env.CARGO_ARGS }} -p ${{ matrix.runtime }}
+
+  zombienet-tests:
+    name: Zombienet tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        binary: [ bajun-node ]
```

Comment:
Note the renaming, but I think this is correct with the new repo structure: https://github.com/ajuna-network/bajun-parachain/pull/26/commits/2207be7bcffef79f5ea8d34f8317e49b387d78c4